<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WEB_CAM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputCam">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>Input </string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect WebCam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openVideoBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="grayscaleBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show GrayScale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flipBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show flip</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputCam">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
